ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+# temp comment for debug
```

Comment:
can you remove this?

Reply:
yes, I needed this just to run the rehearsal.
I think I'll apply Yuval's suggestion and let it re-run again

Reply:
you can just commit --amend to change the commit sha, without applying any fake changes

Reply:
I just learned it first time with this PR, it's necessary to introduce a diff in the `telco5g-cnf-tests-commands.sh`
to open the option to run the rehearsal for the cnf-tests.
see this above: https://github.com/openshift/release/pull/43350#issuecomment-1719219112